lib/console.js

```diff
@@ -62,6 +67,7 @@ function Console(stdout, stderr, ignoreErrors = true) {
   Object.defineProperty(this, '_stderrErrorHandler', prop);
 
   this[kCounts] = new Map();
+  this[kColorMode] = colorMode;
```

Comment:
The constructor should check for invalid values before assigning.

Reply:
:+1: done!